commons/src/main/scala/dagr/commons/reflect/ReflectionUtil.scala

```diff
@@ -225,6 +241,11 @@ object ReflectionUtil {
     enumClass.getDeclaredField(term.name.toString).get(null)
   }
 
+  /** Converts a type tag to a class */
+  def typeTagToClass[T: TypeTag]: Class[T] = {
```

Comment:
Nice!  My personal preference, and you can disagree, is to write methods like this as:

```scala
def typeTagToClass[T: TypeTag]: Class[T] = typeToClass(typeTag[T].tpe).asInstanceOf[Class[T]]
```